python/cudf/cudf_pandas_tests/test_cudf_pandas_fallback.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from cudf.pandas import LOADED
+
+if not LOADED:
+    raise ImportError("These tests must be run with cudf.pandas loaded")
+
+import pandas as xpd
+import pandas._testing as tm
+
+pd = xpd._fsproxy_slow
+
+pytestmark = pytest.mark.filterwarnings(
```

Comment:
This line makes all tests that return the warning fail.

Reply:
Instead of globally configuring this, we could decorate each test with `@pytest.mark.filterwarnings("error::UserWarning:.*Falling back to the slow path.*")` and for test we expect to fallback we could decorate with
```
@pytest.mark.xfail(strict=True, reason="Fallback is expected because...")
@pytest.mark.filterwarnings("error::UserWarning:.*Falling back to the slow path.*")
```